lib/rucio/tests/test_bin_rucio.py

```diff
@@ -2327,3 +2330,20 @@ def test_admin_rse_update_unsupported_option(self):
         print(out, err)
         assert exitcode == 0
         assert not err
+
+    @pytest.mark.noparallel(reason='Modify config')
+    def test_lifetime_cli(self):
+        """ CLIENT(USER): Check CLI to declare lifetime exceptions """
+        # Setup data for CLI check
+        tmp_dsn_name = 'container' + rse_name_generator()
+        tmp_dsn_did = self.user + ':' + tmp_dsn_name
+        self.did_client.add_did(scope=self.user, name=tmp_dsn_name, did_type='DATASET')
+        self.did_client.set_metadata(scope=self.user, name=tmp_dsn_name, key='eol_at', value=(datetime.utcnow() - timedelta(days=1)).strftime('%Y-%m-%d'))
+        self.config_client.set_config_option(section='lifetime_model', option='cutoff_date', value=(datetime.utcnow() + timedelta(days=1)).strftime('%Y-%m-%d'))
+        string = f"%s{tmp_dsn_did}\n"
+        with tempfile.NamedTemporaryFile(mode="w+") as fp:
+            fp.write(string)
+            fp.write(string)
+            fp.seek(0)
+            exitcode, out, err = execute("rucio add-lifetime-exception --inputfile %s --reason 'For testing purpose; please ignore.' --expiration 2124-01-01" % fp.name)
+            assert err.find('does not exist') == -1
```

Comment:
That’s not very Pythonic. How about `assert 'does not exist' not in err`?

Reply:
Done